keywords.txt

```diff
@@ -6,7 +6,7 @@
 # Datatypes (KEYWORD1)
 #######################################
 
-MidiUSB	KEYWORD1
+MIDIUSB	KEYWORD1
```

Comment:
Isnt there still a tab missing? At least on github it looks like there is no tab at all.